public/index.html

```diff
@@ -309,8 +309,8 @@ <h2 id="faq">FAQ</h2>
           </details>
 
           <details class="FAQItem" id="how-does-the-test-work">
-            <summary><span>How does the test work?</span></summary>
-            <p>In order to test if your ISP is implementing BGP safely, we announce a legitimate route but we make sure the announcement is <em>invalid</em>. If you can load the website we host on that route, that means the invalid route was accepted by your ISP. A leaked or a hijacked route would likely be accepted too.</p>
+            <summary><span>Does my ISP support RPKI?</span></summary>
+            <p>In order to test if your ISP is implementing BGP safely, we announce a legitimate route but we make sure the announcement is <em>invalid</em>. If you can load the website we host on that route, that means the invalid route was accepted by your ISP. A leaked or a hijacked route would likely be accepted too. To verify whether your ISP supports RPKI, hit 'Test your ISP' at the top of this website.</p>
```

Comment:
Depending on where we go with this PR, this may become moot, but for completeness of review, we should be using curly double quotes here, so `“Test your ISP”`. A great type designer, Matthew Butterick, has [some thoughts](https://practicaltypography.com/straight-and-curly-quotes.html) on this.